### PR TITLE
Add 'Community creation' warning

### PR DESCRIPTION
**Pull request by nekromant**
I have made a few mini bles for me and friends based on the available specs. As @limpkin suggested I used serial numbers >= 100000 for them. To avoid any frustration (in case any of the devices end up on ebay), I suggest treating any devices with serial number >=100000 as 'community creations'. For them: 

- Security challenge is disabled to avoid any irrelevant spam to the (unexpecting) original team,
- A warning is printed that the device is not affiliated with the original team. 

The translation files are updated yet again, I've added relevant Russian translation for the new messages and (hopefully) not too ugly German one. (The latter one needs some checking by some native speaker though, my knowledge of German is pretty bad).

### CODE REVIEW

File: src/MainWindow.ui [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>946</width>
-    <height>632</height>
+    <height>1081</height>
```
Comment (limpkin) [inline src/MainWindow.ui:10]: i think that's a problem.... @deXol ?

File: src/MainWindow.ui [hunk 2]
```diff
@@ -576,8 +576,8 @@ Hint: keep your mouse positioned over an option to get more details.</string>
             <rect>
              <x>0</x>
              <y>0</y>
-             <width>770</width>
-             <height>882</height>
+             <width>932</width>
+             <height>1832</height>
```
Comment (limpkin) [inline src/MainWindow.ui:580]: tagging @deXol too

File: src/MainWindow.ui [hunk 3]
```diff
@@ -3939,8 +3955,8 @@ Hint: keep your mouse positioned over an option to get more details.</string>
               <rect>
                <x>0</x>
                <y>0</y>
-               <width>646</width>
-               <height>885</height>
+               <width>524</width>
+               <height>704</height>
```
Comment (limpkin) [inline src/MainWindow.ui:3959]: here too...

**Comment by limpkin** [comment]
thanks a lot @nekromant !!! It's really appreciated.

File: src/MainWindow.ui (hunk 1 above)
Comment (nekromant) [reply src/MainWindow.ui:10]: I don't remember explicitly changing these sizes, must be qt creator's doing. Will double-check tomorrow.

**Comment by nekromant** [comment]
Obligatory screenshot of how it looks like with a homebrew device: 

Screenshot_20240918_233838

File: src/MainWindow.ui (hunk 1 above)
Comment (nekromant) [reply src/MainWindow.ui]: Fixed

File: src/MainWindow.ui (hunk 2 above)
Comment (nekromant) [reply src/MainWindow.ui]: fixed

File: src/MainWindow.ui (hunk 3 above)
Comment (nekromant) [reply src/MainWindow.ui]: also fixed

**Comment by limpkin** [comment]
@deXol could you review this one? :)

**Review by deXol — APPROVED** [review]

**Pull request merged**